selfdrive/ui/translations/main_ja.ts

```diff
@@ -408,7 +408,7 @@ location set</source>
         <location filename="../qt/offroad/networking.cc" line="87"/>
         <location filename="../qt/offroad/networking.cc" line="97"/>
         <source>for &quot;</source>
-        <translation>のため &quot;</translation>
+        <translation>&quot; のため</translation>
```

Comment:
We should make this an entire sentence in networking.cc, instead of translating each part of the sentence separately, the translation really breaks down here.

Reply:
Yeah, you can do things like automatic plural handling, I'll refactor that